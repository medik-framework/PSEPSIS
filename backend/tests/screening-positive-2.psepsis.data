{
  "HR": 75,
  "BP Sys": 100,
  "Temp": "<395,10>Rat",
  "Pulse Quality": null,
  "Behavior": null,
  "Capillary Refill": null,
  "Skin Color": null
}
