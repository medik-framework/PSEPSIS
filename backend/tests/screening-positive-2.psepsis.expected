[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get age"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get weight"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmWeightEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get high risk conditions"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmHighRiskConditionsEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": [
      "get 3 bucket measurements"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "HR"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "BP Sys"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Temp"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Pulse Quality"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Behavior"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Capillary Refill"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Skin Color"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "SepsisDiagnosis",
    "args": [
      true
    ]
  }
]
